aip/general/0180.md

```diff
@@ -86,6 +86,15 @@ add the new component, but **must not** remove the existing one. In situations
 where this can allow users to specify conflicting values for the same semantic
 idea, the behavior **must** be clearly specified.
 
+### Moving components between files
+
+Components **must not** be moved between files.
+
+Even though moving a component from one proto file to another proto file within
```

Comment:
Suggested edit:

Moving a component from one proto file to another within the same package is wire compatible, however the code generated for languages like C++ or Python will result in breaking change since `import` and `#include` will no longer point to the correct code location.

Reply:
Done!